Sources/AutoLayout/Constrain.swift

```diff
@@ -17,21 +17,20 @@ public final class ConstraintGroup {
     public init() { }
 
     fileprivate var constraints: [NSLayoutConstraint] = [] {
-        willSet { uninstall(constraints) }
-        didSet { install(constraints) }
+        willSet { uninstall() }
     }
 
     public var isActive: Bool {
         get { constraints.allSatisfy { $0.isActive } }
-        set { newValue ? install(constraints) : uninstall(constraints) }
+        set { newValue ? install() : uninstall() }
     }
 
-    private func install(_ constraints: [NSLayoutConstraint]) {
+    private func install() {
```

Comment:
Since we're touching these, perhaps we could use the underlying UIKit naming and call these `activate`/`deactivate`? 💅

Reply:
Fine by me 🙂, since the naming will be even more consistent